src/lib/hooks/use-best-quote/utils/issuance/issuance-quote.ts

```diff
@@ -27,6 +27,7 @@ interface IssuanceQuoteRequest {
   slippage: number
 }
 
+// In scope?
```

Comment:
@janndriessen this file along with `redemption-quote.ts` both are hardcoded to use `chainId: 1`, are there plans to eventually support other chains here, or is it out of scope?

Reply:
I'd leave these out for now - as I don't think they will change anytime soon. But reminded me to add this ticket: https://github.com/IndexCoop/index-app/issues/1321. Additionally, we could think about moving these to the `/quote` endpoint as well some time?